app/assets/stylesheets/admin/dashboard.css.scss

```diff
@@ -222,6 +222,59 @@
   border-bottom: 2px solid #aaa;
 }
 
+.panel-boundary-bottom {
+  padding-top: 5px;
+  margin-left: 5px;
+  margin-right: 5px;
+  border-bottom: 2px solid #aaa;
+}
+
 .book-title-table {
   max-width: 600px;
 }
+
+.panel-title {
+  margin-left: 10px;
+  font-size: 23px;
+  margin-top: 5px;
+  padding-bottom: 5px;
+}
+
+.book-search-panel-left {
+  width: 49%;
```

Comment:
Properties should be ordered border-right, float, margin-bottom, margin-top, padding-right, width